### DISCUSSION TITLE
folke/trouble.nvim - lazy loading problem, and alternatives?

### DISCUSSION THREAD

**Discussion by dstromberg** [LunarVim/LunarVim#3683] [General]
Hi folks.

Is there a way to get folke/trouble.nvim to work in LunarVim?  Folke appears to believe it's not working because of lazy loading in LunarVim:
https://github.com/folke/trouble.nvim/issues/245
What I'm seeing is that Trouble shows no signs of having been loaded: no errors, nothing in checkhealth, no :trouble command.

Alternatively, is there another way of getting a summary of LSP-identified problems in a given file at the bottom of the screen, sort of like syntastic does for plain vim?

Thanks!

**Comment by LostInTheLogs** [comment]
you're not supposed to use packer directly in your config.lua https://www.lunarvim.org/docs/plugins#user-plugins

**Reply by dstromberg** [reply]
Thank you.  That solved the problem.  I now have the following:
```
lvim.plugins = {
  {'ibhagwan/fzf-lua'},
  {"folke/trouble.nvim"},
}
```
...in my .config/lvim/config.lua

^_^

**Reply by simon-liebehenschel** [reply]
@dstromberg Sorry for touching a closed thread, but do you know where to put `require("devcontainer").setup{}`?

I've read LunarVim and Packer documentation but I still do not understand how to install some plugins that required writing `require(...`

**Reply by LostInTheLogs** [reply]
@AIGeneratedUsername 
```lua
lvim.plugins = {
  {
  'the_devcontainer_plugin_author/plugin_name',
  config = function()
    require("devcontainer").setup{}
  end
  },
}
